compiler/src/dotty/tools/dotc/transform/localopt/InlineLocalObjects.scala

```diff
@@ -34,11 +34,11 @@ class InlineLocalObjects(val simplifyPhase: Simplify) extends Optimisation {
   val gettersCalled = mutable.HashSet[Symbol]()
 
   // Map from class to new fields, initialised between visitor and transformer.
-  var newFieldsMapping: Map[Symbol, Map[Symbol, Symbol]] = null
-  //                   |           |       |
-  //                   |           |       New fields, replacements these getters
-  //                   |           Usages of getters of these classes
-  //                   ValDefs of the classes that are being torn apart; = candidates.intersect(gettersCalled)
+  var newFieldsMapping: Map[Symbol, List[(Symbol, Symbol)]] = null
```

Comment:
Aren't these fields supposed to be private?

Reply:
I didn't bother annotating everything private, but yes, everything is `private[transform.localopt]` minimum.